src/providers/Password.ts

```diff
@@ -107,9 +119,19 @@ export function Password<DataModel extends GenericDataModel>(
   return ConvexCredentials<DataModel>({
     id: "password",
     authorize: async (params, ctx) => {
+      const flow = params.flow as string;
+      if (flow === "signUp" || flow === "reset-verification") {
+        const password = (
```

Comment:
nit: took me a bit to convince myself that this logic was equivalent to what we had before
```
const passwordToValidate = flow === "signUp" ? params.password : flow === "reset-verification" ? params.newPassword : null
```
makes a little more sense to me

Reply:
Sure, that seems fine.